core/tests/test_utils.py

```diff
@@ -164,10 +195,21 @@ def signup_superadmin_user(self):
     def log_line(self, line):
         """Print the line with a prefix that can be identified by the
         script that calls the test.
+
+        Args:
+            line: str. Contains log information which will be
+            prepended with LOG_LINE_PREFIX for display.
         """
         print '%s%s' % (LOG_LINE_PREFIX, line)
 
     def _delete_all_models(self):
+        """Delete entities of the test environment
```

Comment:
`Deletes all datastore entities in the test environment. (Implemented in subclasses.)`